fuzz/Cargo.toml

```diff
@@ -17,12 +17,14 @@ virtio-devices = { path = "../virtio-devices" }
 vmm-sys-util = ">=0.3.1"
 vm-virtio = { path = "../vm-virtio" }
 vm-memory = "0.2.1"
+seccomp = { git = "https://github.com/firecracker-microvm/firecracker", tag = "v0.22.0" }
```

Comment:
@rbradford I am not sure about the use of `[dependencies]` and `[patch.crates-io]`. Naively, I put the direct dependency as `[dependencies]` and  the in-direct dependency as `[patch.crates-io]`. Any inputs?

Reply:
Both of these should be in dependencies. The patch is if we need to override a dependency from a crate we depend upon.

Try and keep these lists lexically sorted fixing any that are wonky.

Adding the action sounds like a good idea!

Reply:
@rbradford Fixed. Thanks for the explanation. I will add the GH action with a different PR.